observ/watcher.py

```diff
@@ -95,11 +103,7 @@ def traverse(obj, seen=None):
         seen = []
     seen.append(obj)
     for v in val_iter:
-        if (
-            isinstance(
-                v, (dict, DictProxyBase, list, ListProxyBase, set, SetProxyBase, tuple)
-            )
-        ) and v not in seen:
+        if v not in seen:
```

Comment:
Wasn't there a reason that we checked the type here? Now that I'm thinking about it: if it would have been important I think we would have added a regression test, I hope 😅

Reply:
Well the thing is I have to add `object` to the list now, and everything in python is an object, so the test becomes useless.